README.md

```diff
@@ -8,10 +8,10 @@ A c library implementing the POST API setup method for general-purpose CPUs and
 
 ## Runtime System Requirements
 
-Windows 10, macOS or Ubuntu.
+Windows 10/11, macOS or Ubuntu.
 One or more of the following processors:
 
-- A GPU and drivers with CUDA support (minimum compute compatibility 5.0, maximum compute compatibility 8.6), such as a modern Nvidia GPU and Nvidia drivers version R450 or newer.
+- A GPU and drivers with CUDA support (minimum compute compatibility 5.0, maximum compute compatibility 9), such as a modern Nvidia GPU and Nvidia drivers version R525 or newer.
```

Comment:
```suggestion
- A GPU and drivers with CUDA support (minimum compute compatibility 5.0, maximum compute compatibility 9), such as a modern Nvidia GPU and Nvidia drivers [version R525](https://www.nvidia.com/Download/driverResults.aspx/194278/en-us/) or newer.
```

Reply:
Well for some of the platforms (Ubuntu) you have dedicated tools to do that, not sure if that makes sense. For Arch you have wiki page etc.

Imho makes no sense to add the link here.

Reply:
This is pretty good, I used this guide, it explains all the options: https://www.cyberciti.biz/faq/ubuntu-linux-install-nvidia-driver-latest-proprietary-driver/